### ISSUE TITLE
[APP SUBMITTED]: 17:43:33 ERROR::FINDSUBTITLES :: [89a4599] No show id found for u'** 403: Series Not Permitted **' ({'country': None, 'year': 2016})
None

### ISSUE THREAD

**Issue by duramato** [pymedusa/SickRage#299]
### INFO
Python Version: **2.7.6 (default, Jun 22 2015, 18:00:18) [GCC 4.8.2]**
Operating System: **Linux-3.19.0-56-generic-i686-with-Ubuntu-14.04-trusty**
Locale: UTF-8
Branch: **develop**
Commit: PyMedusa/SickRage@89a459923cfb2020bc32aab80527bc8a263731d5
No Log available with ERRORS:
### ERROR
```
17:43:33 ERROR::FINDSUBTITLES :: [89a4599] No show id found for u'** 403: Series Not Permitted **' ({'country': None, 'year': 2016})
None
```
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by ratoaq2** [comment]
@duramato, can you give us an example of one release/file name for this show?

**Comment by duramato** [comment]
I dont know, but thats a show that is not allowed on tvdb and got removed

**Comment by ratoaq2** [comment]
It seems that those logging levels will not be changed from ERROR to a lower level as you can see here: https://github.com/Diaoul/subliminal/issues/619

I would like to avoid patching subliminal. Ideal scenario is to use subliminal as it is. Any suggestions?

**Comment by ratoaq2** [comment]
This was fixed by #310 
These refiners errors are logged as info; warnings are logged as debug.
